tests/Library/Vanilla/EmbeddedContent/Embeds/TwitchEmbedFilterTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * @copyright 2009-2019 Vanilla Forums Inc.
+ * @license GPL-2.0-only
+ */
+
+namespace VanillaTests\Library\EmbeddedContent\Embeds;
+
+use Vanilla\EmbeddedContent\Embeds\TwitchEmbed;
+use Vanilla\EmbeddedContent\Embeds\TwitchEmbedFilter;
+use Vanilla\EmbeddedContent\Factories\TwitchEmbedFactory;
+use VanillaTests\Fixtures\MockHttpClient;
+use VanillaTests\MinimalContainerTestCase;
+
+/**
+ * Test twitch embed filter.
+ */
+class TwitchEmbedFilterTest extends MinimalContainerTestCase {
+
+    /** @var TwitchEmbedFactory */
+    private $factory;
+
+    /** @var MockHttpClient */
+    private $httpClient;
+
+    /** @var \Gdn_Request */
+    private $request;
+    
+    /**
+     * Set the factory and client.
+     */
+    public function setUp(): void {
+        parent::setUp();
+        $this->httpClient = new MockHttpClient();
+        $this->factory = new TwitchEmbedFactory($this->httpClient);
+        $this->request = new \Gdn_Request();
```

Comment:
I'm gonna nitpick a little, but let's import this via `use` instead of using the FQN with a leading slash.